loom/graphics/gfxTexture.cpp

```diff
@@ -277,6 +282,8 @@ void bitmapExtrudeRGBA_c(const void *srcMip, void *mip, int srcHeight, int srcWi
 
 TextureInfo *Texture::load(uint8_t *data, uint16_t width, uint16_t height, TextureID id)
 {
+	id &= TEXTURE_ID_MASK;
```

Comment:
We actually do have `TextureInfo* TextureInfo::getTextureInfo(TextureID id)` which checks the id for validity. It's not used everywhere though.